test/chains/chain-tests.js

```diff
@@ -959,6 +959,16 @@ describe("Test Supported Chains", function () {
     "333000333/multicall3.metadata.json"
   );
 
+  // Kiwi Subnet
+  verifyContract(
+    "0xe89a85b79e64b35829625A7EEf70F8915d32F75f",
+    "2037",
+    "Kiwi Subnet",
+    ["multicall3/Multicall3.sol"],
```

Comment:
I think here it should be `2037/multicall3.metadata.json` instead of `multicall3/Multicall3.sol`

Reply:
see the error in the CI